Codecademy/gamut
# feat(ColorMode): add text-secondary + new white weights

### Overview


Add `text-secondary` and new white weights


### PR Checklist

- [x] Related to JIRA ticket: [WEB-2224]

#### Testing Instructions



Check out the `ColorMode` story + see the new `text-secondary` token + examples for the new white color weights.

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |




[WEB-2224]: https://codecademy.atlassian.net/browse/WEB-2224?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
